app/javascript/components/explore/ExploreDisplayTabs.js

```diff
@@ -384,10 +383,24 @@ export default function ExploreDisplayTabs({
               />
             </div>
             }
+            { enabledTabs.includes('images') &&
+              <div className={shownTab === 'images' ? '' : 'hidden'}>
+                <ImageTab
+                  studyAccession={studyAccession}
+                  imageFiles={exploreInfo.imageFiles}
+                  bucketName={exploreInfo.bucket_id}
+                  isCellSelecting={isCellSelecting}
+                  isVisible={shownTab === 'images'}
+                  getPlotDimensions={getPlotDimensions}
```

Comment:
Could you remind me why we pass this function as a parameter, rather than simply importing it in the component?